packages/renderer/src/lib/deployments/DeploymentEmptyScreen.svelte

```diff
@@ -1,7 +1,25 @@
 <script lang="ts">
+import { onDestroy, onMount } from 'svelte';
+
+import type { IDisposable } from '/@api/disposable';
+
 import DeploymentIcon from '../images/DeploymentIcon.svelte';
 import KubernetesEmptyScreen from '../kube/KubernetesEmptyScreen.svelte';
+import { listenResourcePermitted } from '../kube/resource-permission';
+
+let deploymentsPermitted: boolean = $state(true);
+let disposable: IDisposable | undefined;
+
+onMount(async () => {
+  disposable = await listenResourcePermitted('deployments', (permitted: boolean) => {
+    deploymentsPermitted = permitted;
+  });
+});
+
+onDestroy(() => {
+  disposable?.dispose();
+});
```

Comment:
I think that we could simplify by putting this logic into `KubernetesEmptyScreen`, and passing the resource name (`deployments`) to the `KubernetesEmptyScreen` component (and probably replacing property `title` with properties `titleEmpty` and `titleNotPermitted`).